### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Namrata-J**
#2 added alert component
Have created seperate files for js and non-js alert components.
Please review it, thankyou!!

### CODE REVIEW

File: components/alert/js-alerts/js-alerts.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+.et_dismissible-danger-secondary-alert{
+    background-color: var(--lightest-red);
+}
+
+.et_dismissible-warning-secondary-alert{
+    background-color: var(--lightest-yellow);
+}
+
+.et_dismissible-info-secondary-alert{
```
Comment (Raghav888) [inline components/alert/js-alerts/js-alerts.css]: Good naming convention

File: components/alert/js-alerts/js-alerts.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+.et_advanced-dismissible-primary-alert{
+    background-color: var(--black);
+}
+.alert-icon{
+    align-items: center;
+    background: rgba(245, 242, 242, 0.1);
+    display: flex;
+    justify-content: center;
+}
+.alert-message{
```
Comment (Raghav888) [inline components/alert/js-alerts/js-alerts.css]: Follow only one type of naming

**Review by Raghav888 — APPROVED** [review]
Good

**Comment by ultronjs** [comment]
The code look good but please also add a preview link so that we can get a better idea of the component

**Comment by Namrata-J** [comment]
> The code look good but please also add a preview link so that we can get a better idea of the component

Thankyou so much for the review, will also add preview from now!!

File: components/alert/js-alerts/js-alerts.css (hunk 1 above)
Comment (Namrata-J) [reply components/alert/js-alerts/js-alerts.css:60]: okay will look into it!! and will make the required changes, Thankyou so much for the review!!

File: components/alert/js-alerts/js-alerts.css (hunk 2 above)
Comment (Namrata-J) [reply components/alert/js-alerts/js-alerts.css]: okay will look into it!! and will make the required changes, Thankyou so much for the review!!

**Pull request merged**